cosmo_tester/framework/cluster.py

```diff
@@ -80,12 +116,13 @@ def __str__(self):
         return 'Cloudify manager [{}:{}]'.format(self.index, self.ip_address)
 
     @retrying.retry(stop_max_attempt_number=3, wait_fixed=3000)
-    def use(self):
-        self._cfy.profiles.use('{0} -u {1} -p {2} -t {3}'.format(
-                self.ip_address,
-                self._attributes.cloudify_username,
-                self._attributes.cloudify_password,
-                self._attributes.cloudify_tenant).split())
+    def use(self, tenant=None):
```

Comment:
Because when restoring from 3.4 we have to specify a new tenant to restore into, & then we will need to connect to it to check the deployment later